frontend/package.json

```diff
@@ -14,7 +14,8 @@
         "lint": "npm run lint:type && npm run lint:js",
         "start:dev": "vite",
         "build": "tsc --build --clean && vite build",
-        "preview": "vite preview"
+        "preview": "vite preview",
+        "order:css": "stylelint \"src/**/*.scss\" --fix"
```

Comment:
why do we need this command?

Reply:
you can run command locally if you need
`npm run lint:css -- --fix`

Reply:
removed